spec/controllers/users_controller_spec.rb

```diff
@@ -3,17 +3,98 @@
 RSpec.describe UsersController, type: :controller do
   include AuthSupport, CognitoSupport
 
+  let(:user_id) { "3487568374563874" }
+
+  let(:cognito_users) {
+    {:users => [
+        {username: "3487568374563874",
```

Comment:
```suggestion
        {username: user_id,
```
since we have it declared above?